### ISSUE TITLE
[For others] Port to PaperMC

### ISSUE THREAD

**Issue by GavinGoGaming** [ItzDerock/simplevoicechat-music#7]
I've spent the last hour and a half of my life making a somewhat decent port of this mod to PaperMC. I'd love to get permission from @ItzDerock to post this elsewhere (ex. on Modrinth) - ofc with credit back to the original

Some key differences:
- Many features are extremely broken as I really only tested soundcloud search and a few yt songs that seemed to not work.
- Quotes are not needed around search terms (i forgor moment)
- /music bassboost errors if no number is given... oop

I'll keep it updated with newer versions, but for now i've only tested and built it with Paper 1.21 using voicechat-bukkit-2.5.20

You can download the plugin [here](https://github.com/klashdevelopment/mvn/raw/refs/heads/main/plugins/SimpleVoiceChatMusic-1.0.0.jar) - place it into a Paper 1.21 server plugins folder along with the voicechat plugin. There's no tab completion so refer to /music and the original mod's command

TLDR for Derock: If I can get your permission to post this on modrinth that'd be great :)

**Comment by ItzDerock** [comment]
Yeah, go ahead and post it! I'm sure many people will benefit from your work, nice job! All that I ask is for you (and anyone else for that matter) to adhere to the license I've set for this project (Apache-2.0). At some point I do hope to add paper support natively, see #1, but for now this is a good start.

**Comment by GavinGoGaming** [comment]
Thanks! The modrinth post is now under review at [modrinth.com/plugin/simple-voice-chat-music-paper](https://modrinth.com/plugin/simple-voice-chat-music-paper) :)
